config/routes.rb

```diff
@@ -7,14 +7,14 @@
           :defaults => { apiv: 'v2' },
           :apiv => /v2/,
           :constraints => ApiConstraints.new(version: 2, default: true) do
-      constraints(:id => %r{[^\/]+}) do
```

Comment:
I'm fine merging this because it exactly matches what's in Foreman, but I wonder if `%r{[^/]+}` would also work?

Reply:
It should, since it's still accurate than the previous one, but I'd rather sync between core and plugins.